packages/host/src/lib/csi-controller.ts

```diff
@@ -220,6 +220,14 @@ export class CSIController extends TypedEmitter<Events> {
         this.instancePromise = instanceMain();
     }
 
+    async cleanup() {
+        await Promise.all([
+            this.instanceAdapter.cleanup(),
+            this.communicationHandler.cleanup()
+        ]);
+        this.logger.error("Cleanup done...");
```

Comment:
```suggestion
        this.logger.info("Cleanup done.");
```

Reply:
This part will be removed from PR and added as separated fix in other place.